Solar_System_Ana_Lisa_Sutherland_Octo.rb

```diff
@@ -0,0 +1,94 @@
+class Solar_S
```

Comment:
In general don't use underscore (_) in Class names.  Instead naming this `class SolarSystem` would be more appropriate.